src/services/gitHubService.ts

```diff
@@ -67,21 +67,27 @@ class GitHubService {
     }
   }
 
-  public async addReviewersToPr (request: AddPrReviewersModel) : Promise<boolean> {
+  public async addPrReviewers (request: AddPrReviewersModel) : Promise<void> {
     try {
       await this.client.pulls.requestReviewers({
         owner: request.repoOwner,
         pull_number: request.prNumber,
         repo: request.repoName,
-        reviewers: request.reviewers,
-        team_reviewers: request.teamReviewers
+        reviewers: this.determineReviewers(request.reviewers),
+        team_reviewers: this.determineReviewers(request.teamReviewers)
       })
-      return true
     } catch (err) {
       error(`Cannot add reviewers to pull request ${request.prNumber}`)
       error(err)
     }
-    return false
+  }
+
+  private determineReviewers (input: string[]) : string[] | undefined {
+    if (input.length === 0) {
```

Comment:
What is the connection between this and the code in actionService?
In actionService you return an empty string if undefined, here you return undefined if empty string.

Reply:
Actually, I realized that getInput returns an empty string when no input is provided. When split, we're still left with an empty string. So, the code there simply takes that into account. Thanks, unit tests!

Here, in case the array is empty, I want to make it as if the value is not even provided. Undefined is the default value in this particular case for both reviewers and team_reviewers